### PR TITLE
Version 2.31 release

### PR DESCRIPTION
**Pull request by n1tr0-5urf3r**

### CODE REVIEW

**Review by mbacalan — CHANGES_REQUESTED** [review]
Thanks for the PR. I left some comments. It would also be nice to have the filenames in all lowercase for cross platform compatibility.

File: templates/index.html [hunk 1]
```diff
@@ -47,13 +47,17 @@ <h2>Features</h2>
...
+        <li>Multiboxing support</li>
         <li>Show nearest point of interest on map image hover</li>
-        <li>Show the boss you're currently fighting in raids</li>
+        <li>Show the boss you're currently fighting in raids and fractals</li>
+        <li>Language support for en, es, fr, de</li>
       </ul>
     </div>
     <a id="download" href="/download/latest">Download</a>
-    <p>MD5-Sum: b3ace79e72a644a562b58c748a050379</p>
+    <p>MD5-Sum: 591e07517a23d3e8200a2b61cc944f97</p>
```
[the remark this answers is not in the document]
Comment (n1tr0-5urf3r) [reply templates/index.html]: Yes, good point and implemented that :) File needs to be present of course, but might not be a good idea to push the file itself to this repo

File: static/css/style.css [hunk 2]
```diff
@@ -167,4 +171,50 @@ footer {
...
+  background-color: #333; /* Black background color */
+  color: #fff; /* White text color */
+  text-align: center; /* Centered text */
+  border-radius: 2px; /* Rounded borders */
+  padding: 16px; /* Padding */
+  position: fixed; /* Sit on top of the screen */
+  z-index: 1; /* Add a z-index if needed */
+  left: 50%; /* Center the snackbar */
+  bottom: 30px; /* 30px from the bottom */
+}
```
[the remark this answers is not in the document]
Comment (n1tr0-5urf3r) [reply static/css/style.css:187]: True, removed them :D

File: rpcsite.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 app = Flask(__name__)
 
-CLIENT_VERSION = 2.2
+CLIENT_VERSION = 2.31
```
[the remark this answers is not in the document]
Comment (n1tr0-5urf3r) [reply rpcsite.py:10]: Nop, actually 2.31, small fix after releasing 2.3

**Review by mbacalan — APPROVED** [review]

**Pull request merged**